### PR TITLE
Fix Madara was unable to reset Genre’s checkbox

### PR DESCRIPTION
**Pull request by cuong-tran**
Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [X] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [X] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [X] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [X] Have tested the modifications by compiling and running the extension through Android Studio
- [X] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: lib-multisrc/madara/build.gradle.kts [hunk 1]
```diff
@@ -2,7 +2,7 @@ plugins {
     id("lib-multisrc")
 }
 
-baseVersionCode = 35
+baseVersionCode = 36
```
Comment (AwkwardPeak7) [inline lib-multisrc/madara/build.gradle.kts]: not too important so don't bump version

**Review by AwkwardPeak7 — CHANGES_REQUESTED** [review]
rebase on main as well.
also make sure to update it in any source that have overrides related to this.

File: lib-multisrc/madara/build.gradle.kts (hunk 1 above)
Comment (cuong-tran) [reply lib-multisrc/madara/build.gradle.kts]: on another PR earlier, the change was even less significant but @beerpiss said need to bump version no matter how small change is. It's kind of hard to understand :)

**Comment by cuong-tran** [comment]
> also make sure to update it in any source that have overrides related to this.

```kotlin
protected class GenreList(title: String, genres: List<Genre>) : Filter.Group<GenreCheckBox>(title, genres.map { GenreCheckBox(it.name, it.id) })
``` 
It is not overridable or inheritable.

File: lib-multisrc/madara/build.gradle.kts (hunk 1 above)
Comment (AwkwardPeak7) [reply lib-multisrc/madara/build.gradle.kts]: What beer said holds true but Madara is too big to push update every day

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**